### PR TITLE
Adjust known limitation for alter changefeed and cdc queries

### PR DESCRIPTION
**Pull request by kathancox**
Fixes DOC-9412

Adjusted the language for the `alter changefeed` and cdc query doc limitation to make it clear that users can alter a changefeed to switch out the options still, even if that changefeed uses cdc queries.

### CODE REVIEW

File: src/current/_includes/v23.1/known-limitations/alter-changefeed-cdc-queries.md [hunk 1]
```diff
@@ -0,0 +1 @@
+{% if page.name == "alter-changefeed.md" %} `ALTER CHANGEFEED` {% else %} [`ALTER CHANGEFEED`]({% link {{ page.version.version }}/alter-changefeed.md %}) {% endif %} is not fully supported with changefeeds that use {% if page.name == "cdc-queries.md" %} CDC queries. {% else %} [CDC queries]({% link {{ page.version.version }}/cdc-queries.md %}). {% endif %} You can alter the options that a changefeed uses, but you cannot alter the changefeed target tables. [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/83033)
```
Comment (kathancox) [inline src/current/_includes/v23.1/known-limitations/alter-changefeed-cdc-queries.md:1]: This is the only file with new content. All the other file changes are just adding the include link.

**Review by rharding6373 — APPROVED** [review]
Thank you!

**Review by rmloveland — APPROVED** [review]
LGTM!

**Comment by kathancox** [comment]
TFTRs!

**Pull request merged**